rhombus/tests/check.rhm

```diff
@@ -0,0 +1,35 @@
+#lang rhombus
+
+// I used the check to check the check…
+check:
+  check: values(1, [2, 3])
+         ~matches values(a, [b, c])
+  ~completes
+
+macro 'with_err_to_string:
+         $body ...':
+  'block:
+     def old_port = Port.current_error()
+     def new_port = Port.open_output_string()
+     Port.current_error(new_port)
```

Comment:
Unless `parameterize` doesn't work for some reason, using `parameterize` is better than setting and then resetting the parameter value.

Reply:
That's much better—I missed that in the docs.